mergify_engine/github_events.py

```diff
@@ -338,13 +337,13 @@ async def filter_and_dispatch(
 
 
 async def _get_github_pulls_from_sha(
-    client: github.AsyncGithubInstallationClient,
-    repo_name: str,
+    owner_login: github_types.GitHubLogin,
```

Comment:
Would be simpler to pass a GitHubRepository entirely.

Reply:
We don't have one at this point.

Reply:
Fair enough.